app/isomorphic/components/layouts/footer/index.js

```diff
@@ -1,49 +1,31 @@
 import React from "react";
-import PT from "prop-types";
-import { connect } from "react-redux";
-import get from "lodash/get";
-import { Link } from "@quintype/components";
 import "./styles.m.css";
+import logo from "./logo.png";
+import assetify from "@quintype/framework/assetify";
 
-const FooterBase = ({ footerLinks }) => (
-  <div styleName="footer">
-    {footerLinks.map(
-      item =>
-        item.isExternalLink ? (
-          <a
-            href={item.completeUrl}
-            target="_blank"
-            rel="noopener noreferrer"
-            styleName="link"
-          >
-            {item.title}
-          </a>
-        ) : (
-          <Link href={item.completeUrl} styleName="link">
-            {item.title}
-          </Link>
-        )
-    )}
-  </div>
-);
+const Footer = () => {
+  const categories = ["First", "Second", "Third", "Forth", "Fifth"];
+  const sections = ["First", "Second", "Third"];
+  const links = ["First", "Second", "Third", "Forth"];
+  const generateList = (item, id) => <li key={id}>{item}</li>;
 
-function mapStateToProps(state) {
-  return {
-    footerLinks: get(state, ["qt", "data", "navigationMenu", "footerLinks"], [])
-  };
-}
-
-FooterBase.propTypes = {
-  footerLinks: PT.arrayOf(
-    PT.shape({
-      isExternalLink: PT.bool,
-      completeUrl: PT.string,
-      title: PT.string
-    })
-  )
+  return (
+    <div styleName="footer">
+      <img src={assetify(logo)} styleName="logo-footer" alt="Logo" />
+      <div styleName="footer-items">
+        <div styleName="footer-headings">Popular categories:</div>
+        <ul>{categories.map(generateList)}</ul>
+      </div>
+      <div styleName="footer-items">
+        <div styleName="footer-headings">Popular Sections:</div>
+        <ul>{sections.map(generateList)}</ul>
+      </div>
+      <div styleName="footer-items">
+        <div styleName="footer-headings">Quick Links:</div>
+        <ul>{links.map(generateList)}</ul>
+      </div>
```

Comment:
Assuming these too will come from API.
For now, you can add `Quick Links`, `Popular categories` and `Popular Sections` to an array and use map to generate `<div styleName="footer-items">... </div>`. Will reduce code duplication

Reply:
Yeah but two three are used in three different places right? So how can we achieve that
We can do that but might need to change the structure of data but however this is hardcoded data so is that necessary?

Reply:
I haven't modified it but because i am still not aware of how the data is going to be. So even if i assume some structure now and make changes accordingly, we will have to change it again once we start fetching the data through APIs